src/global/text.scss

```diff
@@ -0,0 +1,85 @@
+/**
+ * Typography
+ */
+
+.text-heading-1,
+.text-heading-2,
+.text-heading-3,
+.text-heading-4,
+.text-subheading,
+.text-strong,
+.text-label,
+.text-small {
+  margin-block: 0;
+}
+
+.text-heading-1,
+.text-heading-2,
+.text-heading-3 {
+  font-family: var(--bloom-font-serif);
+  font-weight: normal;
+  line-height: var(--bloom-line-height-heading);
+}
+
+.text-heading-1 {
+  font-size: var(--bloom-type-heading-1);
```

Comment:
I think I would have expected this CSS var to have size somewhere in the title

Reply:
Those specific "type" variable names though _are_ just font size, so maybe we should indicate that...